cirq-core/cirq/neutral_atoms/neutral_atom_devices.py

```diff
@@ -31,6 +30,23 @@ def _subgate_if_parallel_gate(gate: 'cirq.Gate') -> 'cirq.Gate':
     return gate.sub_gate if isinstance(gate, ops.ParallelGate) else gate
 
 
+def neutral_atom_gateset(max_parallel_z=None, max_parallel_xy=None):
```

Comment:
What's the point of keeping this?  Should we fold this into cirq-pasqal as well?

Reply:
This method exists because both the deprecated classes (`NeutralAtomDevice` and `ConvertToNeutralAtomGates`) instantiate a gateset using this method. Once the deprecated classes are removed after deprecation, this method would also be removed. 

Since this method is not exported to the top level cirq modules, I don't think we need to go through a deprecation cycle for this method.